client/state/reader/follows/test/reducer.js

```diff
@@ -367,118 +363,262 @@ describe( 'reducer', () => {
+			const original = deepFreeze( {
+				'example.com': {
+					...exampleFollow,
+					delivery_methods: {
+						email: {
+							send_comments: true,
+						},
+						notification: {},
+					},
+				},
+			} );
+			const state = items( original, unsubscribeToNewCommentEmail( 123 ) );
+			expect( state ).toEqual( {
+				'example.com': {
+					...exampleFollow,
+					delivery_methods: {
+						email: {
+							send_comments: false,
+						},
+						notification: {},
+					},
+				},
+			} );
+		} );
+
+		test( 'should not update when passed identical comment email unsubscription info', () => {
+			const original = deepFreeze( {
+				'example.com': {
+					...exampleFollow,
+					delivery_methods: {
+						email: {
+							send_comments: false,
+						},
+					},
+				},
+			} );
+			const state = items( original, unsubscribeToNewCommentEmail( 123 ) );
+			expect( state ).toEqual( original );
+		} );
+
+		test( 'should not update when passed bad comment email unsubscription info', () => {
+			const original = deepFreeze( {
+				'example.com': {
+					exampleFollow,
+					delivery_methods: {
+						email: {
+							send_comments: true,
+						},
+					},
+				},
+			} );
+			const state = items( original, unsubscribeToNewCommentEmail( 456 ) );
+			expect( state ).toEqual( original );
+		} );
+
+		test( 'should update when a new post notification subscription is received', () => {
 			const original = deepFreeze( {
 				'example.com': {
 					is_following: true,
 					blog_ID: 123,
 					URL: 'http://example.com',
 					delivery_methods: {
 						email: {
-							send_comments: true,
+							send_posts: false,
+						},
+						notification: {
+							send_posts: false,
+						},
+					},
+				},
+			} );
+			const state = items( original, subscribeToNewPostNotifications( 123 ) );
+			expect( state ).toEqual( {
+				'example.com': {
+					is_following: true,
+					blog_ID: 123,
+					URL: 'http://example.com',
+					delivery_methods: {
+						email: {
+							send_posts: false,
+						},
+						notification: {
+							send_posts: true,
 						},
 					},
 				},
 			} );
-			const state = items( original, subscribeToNewCommentEmail( 123 ) );
-			chaiExpect( state ).to.equal( original );
 		} );
 
-		test( 'should not update when passed comment sub info about a missing sub', () => {
+		test( 'should not update when a new post notification subscription is identical', () => {
 			const original = deepFreeze( {
 				'example.com': {
 					is_following: true,
 					blog_ID: 123,
 					URL: 'http://example.com',
 					delivery_methods: {
 						email: {
-							send_comments: true,
+							send_posts: true,
+						},
+						notification: {
+							send_posts: true,
 						},
 					},
 				},
 			} );
-			const state = items( original, subscribeToNewCommentEmail( 456 ) );
-			chaiExpect( state ).to.equal( original );
+			const state = items( original, subscribeToNewPostNotifications( 123 ) );
+			expect( state ).toEqual( original );
```

Comment:
these "should not update" tests should use `toBe` instead of `toEqual` to make sure the object instance isn't changing.